app.json

```diff
@@ -25,11 +25,12 @@
   },
   "addons": [
     "heroku-redis",
-    "scheduler"
+    "scheduler",
+    "sentry:f1"
```

Comment:
@amureki is this actually the plan or a typo?

Reply:
![image](https://user-images.githubusercontent.com/627183/37211260-a8195990-23ab-11e8-9e36-d7d0f65642bb.png)

it is a free plan :)